domino_data/configuration_gen.py

```diff
@@ -239,6 +240,17 @@ class TrinoConfig(Config):
     password: Optional[str] = _cred(elem=CredElem.PASSWORD)
 
 
+@attr.s(auto_attribs=True)
+class TabularS3GlueConfig(Config):
+    """TabularS3GlueConfig datasource configuration."""
+
+    database: Optional[str] = _config(elem=ConfigElem.DATABASE)
+    region: Optional[str] = _config(elem=ConfigElem.REGION)
+
+    aws_access_key_id: Optional[str] = _cred(elem=CredElem.ACCESSKEYID)
+    aws_secret_access_key: Optional[str] = _cred(elem=CredElem.SECRETACCESSKEY)
```

Comment:
those fields seems wrong TBH, pretty sure almost nothing should be overridable for TabularS3

Reply:
mind sharing how you determine a field should be overridable? i just copied what was there for normal S3.

Reply:
First whether they can actually be changed at runtime, second if it makes sense (more of a case by case). Those would require a Trino restart to take effect so they really can't be overridable.